### ISSUE TITLE
Repost displays the_title

### ISSUE THREAD

**Issue by chrisaldrich** [dshanske/indieweb-post-kinds#68]
It's likely an oversight, but reposts are currently displaying the_title (similar to the "article" kind). I suspect that like many other post kinds (eg. note) you probably meant to strip the title out of the display and inject it using "title" in Post Properties.

**Comment by miklb** [comment]
My request may be similar.

If I create a quote kind and retrieve a tweet url, it adds `Jane Doe on Twitter` as the title, which then would be used for the quote content instead of `the_content`, unless I explicitly remove the title before publishing.

A filter to explicitly remove the_title on specific kinds might be sufficient, if you can point me in the direction where to add it, a pull request would be forthcoming.

An example of what I mean, [post on my site](https://miklb.com/afp-news-agency-on-twitter/) and the [POSSE'd tweet](https://twitter.com/miklb/status/854386854451445760).

**Comment by dshanske** [comment]
Stripping the title is a theme issue, but let me think

**Comment by miklb** [comment]
@dshanske I agree, but if the title generated isn't really representative of the content, then perhaps better to not have generate one. Granted, the user is still ultimately responsible for their content, just looking at expected behavior vs result.

**Comment by chrisaldrich** [comment]
<a href="https://github.com/dshanske/indieweb-post-kinds/issues/68#issuecomment-295031896" class="u-in-reply-to"></a>I view replies to be similar to notes in not needing/requiring titles. 

For metadata it can often be useful to have an actual title in the admin UI to identify the post in those administrative views.

What I find even more interesting/useful is having the ability to put in a custom title for automatic POSSE purposes, and the title is often used by many (most? all??) POSSE plugins for just this. In this UX/UI case it can mean some of the content could be displayed multiple times, once in the title and again in the body, which isn't ideal. 

I'm certainly in the boat that stripping the title is a theme issue. I'd argue that for the best possible broad usage, Post Kinds should strip it out (using CSS) as a default and allow those who'd prefer to display it on the front end, just untoggle the CSS setting.

If this is the case, then the final consideration is the duplication of content between the title and body within RSS as well.

**Comment by dshanske** [comment]
When I hid it using CSS, people kept complaining. I removed that to rethink it.

**Comment by miklb** [comment]
Question @dshanske if I retrieve info on a url while set to a note, it doesn't create a title. Where in the code is that decision made? I'd like to read it closer. thanks.

**Comment by miklb** [comment]
Bringing this back up after looking at the new properties metaboxes.

1. Choose bookmark kind
2. add url
3. retrieve the properties.
4. Clear properties.
5. Switch to Article
6. Observe the title from scraped bookmark in the Article title field
